src/main/java/org/orecruncher/dsurround/effects/blocks/producers/BlockEffectProducer.java

```diff
@@ -97,10 +97,8 @@ public static int countCubeBlocks(final World provider,
 
     @Override
     public String toString() {
-        StringBuilder builder = new StringBuilder();
-        builder.append(this.getClass().getSimpleName())
-                .append("{").append("chance: ").append(this.chance)
-                .append("; conditions: ").append(this.conditions).append("}");
-        return builder.toString();
+        return this.getClass().getSimpleName()
+                + "{" + "chance: " + this.chance
+                + "; conditions: " + this.conditions + "}";
```

Comment:
Why does this separate `{` and `chance` but not `;` and `conditions:`? It feels inconsistent in style.